### ISSUE TITLE
Support curses (but has no compatibility)

### ISSUE THREAD

**Issue by steelwheels** [steelwheels/JSTerminal#3]
Update KiwiControl framework to support curses on terminal.

**Comment by steelwheels** [comment]
Prepare to support:
````
commit d60a1c9fd43616880956bc4838b7c53bbc827276
Author: Steel Wheels Project <wheel@tcn.zaq.ne.jp>
Date:   Sun Dec 8 10:17:09 2019 +0900

    CoconutShell: Prepare to support curses
````

**Comment by steelwheels** [comment]
Start defining theJavaScript API for curses.
````
commit 287ecae69045688f97a22a8dee4b4393770ddc3a
Author: Steel Wheels Project <wheel@tcn.zaq.ne.jp>
Date:   Sun Dec 15 22:34:07 2019 +0900

    KiwiLibrary: Update KLCurses class
````

**Comment by steelwheels** [comment]
Update curses support by following commits:
````
commit bfb60501e7d30a9342da85b0869d80fdbf749d6d
Author: Steel Wheels Project <wheel@tcn.zaq.ne.jp>
Date:   Tue May 19 23:26:00 2020 +0900

    KCTextViewCore: Make event when the text size is changed
````

**Comment by steelwheels** [comment]
Implemented into v1.1.
